src/ru/nsu/ccfit/zuev/osu/game/GameObject.java

```diff
@@ -42,4 +43,8 @@ public int getId() {
     public void setId(int id) {
         this.id = id;
     }
+
+    public boolean startHit(){
+        return startHit;
+    }
```

Comment:
请遵循命名规范，对于boolean类型的属性请使用如下的方法
```suggestion

    public boolean isStartHit(){
        return startHit;
    }
```